lib/bamboo/test.ex

```diff
@@ -159,33 +157,46 @@ defmodule Bamboo.Test do
       unsent_email = Bamboo.Email.new_email(subject: "something else")
       assert_delivered_email(unsent_email) # Will fail
   """
-  def assert_delivered_email(%Bamboo.Email{} = email) do
-    email = normalize_for_testing(email)
-    do_assert_delivered_email(email)
+  defmacro assert_delivered_email(email) do
+    quote do
+      import ExUnit.Assertions
+      email = Bamboo.Test.normalize_for_testing(unquote(email))
+      assert_receive({:delivered_email, ^email}, 100, Bamboo.Test.flunk_with_email_list(email))
+    end
   end
 
-  defp do_assert_delivered_email(email) do
-    receive do
-      {:delivered_email, ^email} -> true
-    after
-      @timeout -> flunk_with_email_list(email)
+  @doc """
+  Checks whether an email its params equal to the ones provided.
```

Comment:
One more small typo: Check whether an email's params are equal to the ones provided